rofl-containers/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rofl-containers"
-version = "0.3.3"
+version = "0.3.5"
```

Comment:
Why 5 and not 4?

Reply:
We published 0.3.4 (see tags) but we forgot to bump the crate version.